src/pages/app-contact/app-contact.tsx

```diff
@@ -7,6 +7,43 @@ import { MatchResults } from '@stencil/router';
 })
 export class AppContact {
   @Prop() match: MatchResults;
+  @State() value: string;
+
+  handleSubmit(e) {
+    e.preventDefault();
+
+    const formData = {
+      name: e.target[0].value,
+      email: e.target[1].value,
+      phone: e.target[2].value,
+      message: e.target[3].value,
+      desiredService: e.target[4].value,
+      budget: e.target[5].value,
+    };
+
+    fetch(
```

Comment:
I think that using async/await for request its more cleaner and readable. You have an example here https://gist.github.com/msmfsd/fca50ab095b795eb39739e8c4357a808 😊

Reply:
I agree that async/await is more readable.

But regardless, we should also be showing some sort of feedback to the user whether the form has been successfully submitted or an error occurred during transmission